### PR TITLE
create community page

### PR DESCRIPTION
**Pull request by francisco-core**
Page that displays how people can join the community

It looks like this:

## Desktop
Screenshot-2019-8-29 Communities(2)

## Mobile
Screenshot-2019-8-29 Communities(1)

close issue https://github.com/PrivacyLx/privacylx-issue-tracker/issues/41

### CODE REVIEW

File: themes/hugo-bootstrap/i18n/en.yaml [hunk 1]
```diff
@@ -97,3 +97,54 @@
...
+  translation: "We are a community of people who deeply care about privacy and consider it fundamental to a free society"
+
+- id: Forum
+  translation: "Forum"
+
+- id: "join our forum"
+  translation: "Where we have lively discussions about privacy and surveillance"
+
+- id: "all are welcome"
+  translation: "Todos s  o bem-vindos independentemente do conhecimento que tenham j   sobre o t  pico."
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/en.yaml]: Here there should be instead the English translation of the id.

File: themes/hugo-bootstrap/i18n/en.yaml [hunk 2]
```diff
@@ -97,3 +97,54 @@
...
+
+- id: "Chat"
+  translation: "Chat"
+
+- id: "Come and say hello on our chat"
+  translation: "Come and say hello on our chat"
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/en.yaml]: s/on our/to our

File: themes/hugo-bootstrap/i18n/en.yaml [hunk 3]
```diff
@@ -97,3 +97,54 @@
...
+  translation: "Teach"
+
+- id: "teach-description"
+  translation: "As part of our job we provide free education for people to know how they can be protected from surveillance."
+
+- id: "teach-link"
+  translation: "https://cafe.privacylx.org/new-topic?title=Idea%20for%20an%20event%20about...&body=Hello!%20I%20am%20interested%20in%20organizing%20an%20event%20about...&category=event-ideas"
+
+- id: "develop"
+  translation: "Develop"
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/en.yaml]: Help seems better at this case.
We have many issues or tasks that do not require any knowledge of programming languages.

File: themes/hugo-bootstrap/i18n/pt.yaml [hunk 4]
```diff
@@ -96,3 +96,54 @@
...
   translation: "Aqui colocamos recursos que criamos no contexto dos workshops, tais como apresentações, reports e tudo o que possa ser do interesse do leitor."
+
+- id: join-us-description
+  translation: "Somos uma comunidade que valoriza a privacidade como algo fundamental para uma sociedade livre."
+
+- id: join-us
+  translation: "Junta-te a nós!"
+
+- id: "all are welcome"
+  translation: "Todos são bem-vindos independentemente do conhecimento que tenham já sobre o tópico."
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/pt.yaml]: Perhaps: Tod@s são bem-vind@s [...]

File: themes/hugo-bootstrap/i18n/pt.yaml [hunk 5]
```diff
@@ -96,3 +96,54 @@
...
+- id: Forum
+  translation: "Forum"
+
+- id: "join our forum"
+  translation: "O principar meio de comunicação da comunidade"
+
+- id: "Chat"
+  translation: "Chat"
+
+- id: "Come and say hello on our chat"
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/pt.yaml]: s/on our chat/to out chat

File: themes/hugo-bootstrap/layouts/community/list.html [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+       <div class="col-lg-6 text-center">
+	<h2>{{ i18n "Forum" }}</h2>
+	<p>{{ i18n "join our forum" }}</p>
+	<a href="https://chat.privacylx.org"> <img src="/img/forum-screenshot.png" style="width: 100%"></img></a>
+	<br></br>
+        </div>
+       <div class="col-lg-6 text-center">
+        <h2>{{ i18n "Chat" }}</h2>
+        <p>{{ i18n "Come and say hello on our chat" }}</p>
+        <a href="https://riot.im/app/#/group/+privacylx:matrix.org">
```
Comment (anadahz) [inline themes/hugo-bootstrap/layouts/community/list.html]: The correct URL is: https://riot.im/app/#/room/#privacylx:matrix.org

File: themes/hugo-bootstrap/layouts/community/list.html [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+	  <img src="/img/matrix-chat.png" style="width: 100%"></img>
+	</a>
+	{{ i18n "The channel can also be accessed through"}} <a href="https://irc.lc/oftc/privacylx/">{{ i18n "IRC (#privacylx on OFTC)" }}</a>
```
Comment (anadahz) [inline themes/hugo-bootstrap/layouts/community/list.html]: This URL is displaying an empty as mentioned in https://github.com/PrivacyLx/privacylx-issue-tracker/issues/41#issuecomment-526175272 it's better to switch to OFTC's webchat, is uglier and doesn't (usually) permit Tor exits, but it works.

File: themes/hugo-bootstrap/i18n/pt.yaml [hunk 8]
```diff
@@ -96,3 +96,54 @@
...
+  translation: "Para conversas rápidas, em real-time ou pequenas questões este é o sítio"
+
+- id: "The channel can also be accessed through"
+  translation: "The channel can also be accessed through"
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/pt.yaml]: Missing Portuguese translation.

File: themes/hugo-bootstrap/i18n/pt.yaml [hunk 9]
```diff
@@ -96,3 +96,54 @@
...
+  translation: "Ensinar"
+
+- id: "teach-description"
+  translation: "Parte do nosso trabalho consiste em dar formação gratuita sobre o riscos da vigilância digital e como os combater."
+
+- id: "teach-link"
+  translation: "https://cafe.privacylx.org/new-topic?title=Ideia%20para%20evento%20sobre...&body=Boas!%20Estava%20interessado%20em%20organizar%20um%20evento%20sobre...&category=event-ideas"
+
+- id: "develop"
+  translation: "Programar"
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/pt.yaml]: Same as in: https://github.com/PrivacyLx/website/pull/12/files#r319105302

File: themes/hugo-bootstrap/i18n/pt.yaml (hunk 4 above)
Comment (francisco-core) [reply themes/hugo-bootstrap/i18n/pt.yaml]: Good point! But I had already removed that string form the interface but just forgot to remove it here too

File: themes/hugo-bootstrap/i18n/pt.yaml (hunk 8 above)
Comment (francisco-core) [reply themes/hugo-bootstrap/i18n/pt.yaml]: Whoops. Another one I forgot to remove

File: themes/hugo-bootstrap/layouts/community/list.html [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+</div>
+
+<br></br>
+<div class="jumbotron jumbotron-fluid">
+<div class="container">
+  <div class="row">
+       <div class="col-lg-6 text-center">
+	<h2>{{ i18n "Forum" }}</h2>
+	<p>{{ i18n "join our forum" }}</p>
+	<a href="https://chat.privacylx.org"> <img src="/img/forum-screenshot.png" style="width: 100%"></img></a>
```
Comment (francisco-core) [inline themes/hugo-bootstrap/layouts/community/list.html]: Whoops. Major error. chat.privacylx.org instead of cafe.privacylx.org

**Comment by francisco-core** [comment]
It should be ready for merging now

File: themes/hugo-bootstrap/i18n/pt.yaml [hunk 11]
```diff
@@ -96,3 +96,57 @@
...
+  translation: "Somos uma comunidade que valoriza a privacidade como algo fundamental para uma sociedade livre."
+
+- id: join-us
+  translation: "Junta-te a nós!"
+
+- id: Forum
+  translation: "Forum"
+
+- id: "join our forum"
+  translation: "Tod@s são bem-vindos! Este é o principal meio onde nos encontramos."
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/pt.yaml]: Shouldn't bem-vindos supposed to be bem-vind@s?

File: themes/hugo-bootstrap/i18n/pt.yaml (hunk 8 above)
Comment (anadahz) [reply themes/hugo-bootstrap/i18n/pt.yaml]: This should be change to Portuguese.

File: themes/hugo-bootstrap/i18n/pt.yaml [hunk 12]
```diff
@@ -96,3 +96,57 @@
...
+  translation: "Parte do nosso trabalho consiste em dar formação gratuita sobre o riscos da vigilância digital e como os combater."
+
+- id: "teach-link"
+  translation: "https://cafe.privacylx.org/new-topic?title=Ideia%20para%20evento%20sobre...&body=Boas!%20Estava%20interessado%20em%20organizar%20um%20evento%20sobre...&category=event-ideas"
+
+- id: "help"
+  translation: "Ajudar"
+
+- id: "help-description"
+  translation: "Como somos todos voluntários, há uma série de tarefas que estão só à espera da pessoa certa para as fazer."
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/pt.yaml]: s/os/@s

Comment (francisco-core) [reply themes/hugo-bootstrap/i18n/pt.yaml]: Done!

**Comment by francisco-core** [comment]
The typo has been fixed. Shall we merge this? Let's leave the inconsistencies in i18n variable naming aside, please.

**Comment by anadahz** [comment]
There are still some issues with this PR (apart from the inconsistencies with the strings).
Please go through my review comments so that you will not forget anything.

**Comment by anadahz** [comment]
@francisco-core I made the required fixes, please check if everything is fine.

**Pull request merged**